src/Entity/Styles/apa.xml

```diff
@@ -0,0 +1,1920 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What did you change in this file from the original?

Reply:
Alot

Reply:
Document the lot.

Reply:
because of all the changes, i'll document how to see those differences in the readme along side these files. It's just using the `diff` command in the terminal.